app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -345,6 +336,10 @@ public void displayCurrentInfo() {
                 currentBgValueText.setPaintFlags(currentBgValueText.getPaintFlags() | Paint.STRIKE_THRU_TEXT_FLAG);
                 dexbridgeBattery.setPaintFlags(dexbridgeBattery.getPaintFlags() | Paint.STRIKE_THRU_TEXT_FLAG);
             } else {
+                if (notificationText.getText().length()==0){
+                    notificationText.setTextColor(Color.WHITE);
+                    notificationText.setText((System.currentTimeMillis()-lastBgreading.timestamp)/(60*1000) + " Minutes ago");
```

Comment:
thoughts on rearanging this a bit so that

(starting on line 328)
```java
 if ((new Date().getTime()) - (60000 * 11) - lastBgreading.timestamp > 0) {
    notificationText.setText("Signal Missed");
    ...all that other jazz
} else {
     if (notificationText.getText().length() == 0){
        notificationText.setTextColor(Color.WHITE);
    }
}
notificationText.append("\n" + (System.currentTimeMillis() - lastBgreading.timestamp)/(60*1000) + " Minutes ago");
```

that way even if we know the signal has been missed we can get how long ago the last one was